src/workers/UniswapV3LiquidityWorker.ts

```diff
@@ -18,17 +20,25 @@ class UniswapV3LiquidityWorker extends BasicWorker {
 
   apply = async (job: Bull.Job): Promise<void> => {
     if (this.isStale(job)) return;
+    const loggerPrefix = `[UniswapV3LiquidityWorker][${job.data.chainId}][${job.data.protocol}]`;
 
     const {lock} = job.data.settings;
     const unlocked = await this.connection.set(lock.name, 'lock', 'EX', lock.ttl, 'NX');
 
     if (!unlocked) {
-      this.logger.error('[UniswapV3LiquidityWorker] apply for job but job !unlocked');
+      this.logger.error(`${loggerPrefix} apply for job but job !unlocked`);
+      return;
+    }
+
+    const isValidSettings = this.checkIsValidSettings(job.data);
+
+    if (!isValidSettings) {
```

Comment:
this check should be first one and should not display an error - you can log.debug